scripts/lis_hyperv_platform/main.ps1

```diff
@@ -1,12 +1,11 @@
 param(
     [String] $SharedStoragePath = "\\shared\storage\path",
```

Comment:
add sharedstorage username and password as two other params and use it accordingly